src/references/auth.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Authentication
+---
+
+## Authentication Sequence
+
+1. The User opens up a browser to check his cozy, using his *cozy app domain* (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...)
+2. he's redirected to his *cozy user domain* (`user.mycozy.cloud`)
+3. The browser request the *cozy user domain*, which send back the authentication form. The user fills his password, sending it back to the  *cozy user domain*
+4. The server *cozy user domain* set an auth cookie for *cozy user domain* ONLY
```

Comment:
```suggestion
4. The server *cozy user domain* sets an auth cookie for *cozy user domain* ONLY
```

Why ONLY ?

Reply:
This cookie is restricted to this sub-domain (and httpOnly). It will *not* be seen by the app domain.

This is by design, so that an app could not use the cookie and authenticate with another app permission.

Reply:
It's a bit confused. Let's rephrase it:

- we are using `httpOnly` to ensure that the JS can't play with the cookie
- the fact that we have distinct cookies for the user domain and the app domain is not by design (for self-hosted, it is the same cookie that is used)
- a cookie on `alice.mycozy.cloud` is not used by `alice-drive.mycozy.cloud` (it's how the cookies work).

We can put the cookie on level upper, on `.mycozy.cloud`, but it has some issues (a user can't be logged on two cozy instances at the same time, the cookies are sent when a user visit a page shared by link on another instance, etc.). So, it was simpler to use several cookies with a copy mechanism than to deal with those issues.